engine/install/ubuntu.md

```diff
@@ -89,7 +89,7 @@ You can install Docker Engine in different ways, depending on your needs:
 
 - [Install it manually](#install-from-a-package) and manage upgrades manually.
 
-- Use a [convenience scripts](#install-using-the-convenience-script). Only
+- Use a [convenience script](#install-using-the-convenience-script). Only
```

Comment:
Based on the link text (and associated content) this appears to be singular, and if so, should reflect that.

If there are multiple scripts or a desire to reflect that, then an alternative would be:

```suggestion
- Use [convenience scripts](#install-using-the-convenience-script). Only
```

Reply:
@calebcartwright Personally I think your first suggestion is correct here as it leads to a link that explains using a singular script.

```suggestion
- Use a [convenience script](#install-using-the-convenience-script). Only
```